composer.json

```diff
@@ -25,17 +25,20 @@
         }
     },
     "require" : {
-        "php"                : ">=5.3.3"
+        "php"                               : "^5.5 || ^7.0",
+        "php-http/httplug"                  : "^1.0",
+        "php-http/message-factory"          : "^1.0",
+        "php-http/discovery"                : "^1.0",
+        "php-http/multipart-stream-builder" : "^0.1",
+        "php-http/client-implementation"    : "^1.0"
     },
     "require-dev" : {
-        "kriswallsmith/buzz" : "~0.8",
-        "guzzle/http"        : "~3",
-        "guzzlehttp/guzzle"  : "~5.2"
+        "phpunit/phpunit" : "^4.8 || 5.0 - 5.3",
+        "php-http/message" : "^1.0",
+        "php-http/guzzle6-adapter" : "^1.0"
```

Comment:
Yes, or we need a PSR-7 implementation because we use create Requests and Responses (even in tests) 

We do also need a client-implementation. I will replace that line for 
```json

        "guzzlehttp/psr7" : "^1.3",
        "php-http/mock-client" : "^0.3"
```
to be more clear about this.